DOKS-wordpress/assets/manifests/values.yml

```diff
@@ -33,18 +38,4 @@ externalDatabase:
 
 # Disabling MariaDB
 mariadb:
-  enabled: false
-
-# Enable ingress record generation for WordPress
-ingress:
-  enabled: true
-  certManager: true
-  tls: false
-  hostname: <YOUR_WORDPRESS_DOMAIN_HERE>
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
-    cert-manager.io/cluster-issuer: "letsencrypt-prod"
-  extraTls:
-  - hosts:
-      - <YOUR_WORDPRESS_DOMAIN_HERE>
-    secretName: wordpress.local-tls
+  enabled: false
```

Comment:
Make sure that files have new line endings. There's a setting in VS Code which automatically adds a new line to files after saving. 
Open VS Code -> Preferences -> Settings, and then in the top bar search for `insert final newline`:

<img width="1016" alt="Screenshot 2022-05-11 at 09 35 45" src="https://user-images.githubusercontent.com/87807677/167784069-fe65a8ae-1be9-4555-bc67-f83476e41a2f.png">